.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Please don't include any `editor related` files. track vscode only on your system. Not everyone is using vscode and this is not a project related file.

Hint: in vscode you can add dictionary word to your workspace or locally not per project. 

this include changes to `.gitignore`